bin/update_ati_analytics.py

```diff
@@ -0,0 +1,125 @@
+"""
+Gets the ati-informal-requests-analytics.csv from the SMB
+and filters out the current month's rows and then patches the
+Resource on the Registry.
+"""
+
+import os
+import click
+import csv
+import traceback
+import tempfile
+from datetime import datetime
+import subprocess
+from six import PY2
+from shutil import rmtree
+from codecs import BOM_UTF8
+
+if PY2:
+    from cStringIO import StringIO
+else:
+    from io import StringIO
```

Comment:
this is a standalone script, why bother with python2 support?

Reply:
Yeah, I guess we would just need to do `pip3 install click` if we do not already have that dependency on the Registry servers for python 3

Or if that GA venv is still on the servers and is using py3, then that could be a fine venv to use